packages/google_maps_flutter/google_maps_flutter_platform_interface/test/types/bitmap_test.dart

```diff
@@ -185,7 +185,9 @@ void main() {
 
           expect((mip.toJson() as List<dynamic>)[2], 1);
           expect((scaled.toJson() as List<dynamic>)[2], 3);
-        });
+        },
+            // TODO(stuartmorgan): Investigate timeout on web.
```

Comment:
Does this need an issue?

Reply:
Based on a comment from David later I think we just need to change this to inline image data. I'll do a fast-follow on this.